### PR TITLE
Exercise 2 done

### PR DESCRIPTION
**Pull request by mohit-loconav**

### CODE REVIEW

**Comment by mohit-loconav** [comment]
PTAL @gagan93jtg

File: html/Exercise 2/index.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Exercise 2</title>
+
+    <style>
+
+        /* body {
```
Comment (gagan93jtg) [inline html/Exercise 2/index.html]: **Generic Issues**: unrequired spacing, commented lines, 2 space indentation

File: html/Exercise 2/index.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            margin-left: auto;
+            margin-right: auto;
+        } */
+
+        * {
+            font-family: 'Muli',sans-serif;
```
Comment (gagan93jtg) [inline html/Exercise 2/index.html]: Use **Raleway**

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

File: html/Exercise 2/index.html [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100&display=swap" rel="stylesheet">
+  <title>Exercise 2</title>
+
+  <style>
+
+    * {
+      font-family: 'Raleway',sans-serif;
```
Comment (gagan93jtg) [inline html/Exercise 2/index.html:13]: check the design once. Entire design is so less in font weight that I can barely read anything
Screenshot 2021-03-15 at 1 43 11 PM

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

**Review by gagan93jtg — APPROVED** [review]

**Pull request merged**